### PR TITLE
Ptolmic Military Colony

### PR DESCRIPTION
**Pull request by TheShadowOfHassen**

### CODE REVIEW

File: 0-ad-history-mod-package/simulation/templates/structures/ptol/military_colony.xml [hunk 1]
```diff
@@ -8,6 +8,9 @@
     <Civ>ptol</Civ>
     <SpecificName>Klērouchia</SpecificName>
     <Classes datatype="tokens">CivSpecific</Classes>
+    <History>
+        The Ptolemaic empire granted veteran Greek troops land in colonies, in the empire. These pockets of Greek culture spread throughout the traditional Egyptian land giving shape to the Hellenistic kingdom.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/structures/ptol/military_colony.xml:12]:
```suggestion


The Greco-Macedonian pharaohs of the Ptolemaic dynasty granted plots of land to all veteran Greek soldiers who would settle within the borders of the empire. These veterans and their decendants would then be called upon by the pharaoh to serve in the imperial army during times of war or rebellion. The pockets of Greek culture created by these settlements would influence and give shape to the traditional native Egyptian culture, creating a unique syncretism we today call the Hellenistic culture.

```

File: 0-ad-history-mod-package/simulation/templates/structures/ptol/military_colony.xml [hunk 2]
```diff
@@ -8,6 +8,10 @@
     <Civ>ptol</Civ>
     <SpecificName>Klērouchia</SpecificName>
     <Classes datatype="tokens">CivSpecific</Classes>
+    <History>
+    The Greco-Macedonian pharaohs of the Ptolemaic dynasty granted plots of land to all veteran Greek soldiers who would settle within the borders of the empire. These veterans and their decendants would then be called upon by the pharaoh to serve in the imperial army during times of war or rebellion. The pockets of Greek culture created by these settlements would influence and give shape to the traditional native Egyptian culture, creating a unique syncretism we today call the Hellenistic culture.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/structures/ptol/military_colony.xml:12]:
```suggestion
        The Greco-Macedonian pharaohs of the Ptolemaic dynasty granted plots of land to all veteran Greek soldiers who would settle within the borders of the empire. These veterans and their decendants would then be called upon by the pharaoh to serve in the imperial army during times of war or rebellion. The pockets of Greek culture created by these settlements would influence and give shape to the traditional native Egyptian culture, creating a unique syncretism we today call the Hellenistic culture.
```

File: Sources/Ptolemies.md [hunk 3]
```diff
@@ -25,3 +25,5 @@ Lighthouse: https://www.britannica.com/topic/lighthouse-of-Alexandria  by Vantha
 Temples: Daily Life in The Hellenistic Age by James Allan Evans : By ShadowOfHassen
 
 Wonder: https://www.ancient-egypt.co.uk/edfu_main_temple/ & https://www.ancient-egypt.co.uk/edfu_main_temple/ : By ShadowOfHassen
+
+Military Colony: Based of the Selecluid article. https://en.wikipedia.org/wiki/Ptolemaic_Kingdom : By ShadowOfHassen
```
Comment (TheShadowOfHassen) [inline Sources/Ptolemies.md:29]:
```suggestion
Military Colony: Based of the Selecluid article. https://en.wikipedia.org/wiki/Ptolemaic_Kingdom : By WowGetOffYourPhone & ShadowOfHassen
```

File: 0-ad-history-mod-package/simulation/templates/structures/ptol/military_colony.xml [hunk 4]
```diff
@@ -8,6 +8,10 @@
     <Civ>ptol</Civ>
     <SpecificName>Klērouchia</SpecificName>
     <Classes datatype="tokens">CivSpecific</Classes>
+    <History>
+        The Greco-Macedonian pharaohs of the Ptolemaic dynasty granted plots of land to all veteran Greek soldiers who would settle within the borders of the empire. These veterans and their decendants would then be called upon by the pharaoh to serve in the imperial army during times of war or rebellion. The pockets of Greek culture created by these settlements would influence and give shape to the traditional native Egyptian culture, creating a unique syncretism we today call the Hellenistic culture.
+
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/structures/ptol/military_colony.xml:13]:
```suggestion
```

File: 0-ad-history-mod-package/simulation/templates/structures/ptol/military_colony.xml [hunk 5]
```diff
@@ -8,6 +8,9 @@
     <Civ>ptol</Civ>
     <SpecificName>Klērouchia</SpecificName>
     <Classes datatype="tokens">CivSpecific</Classes>
+    <History>
+        The Greco-Macedonian pharaohs of the Ptolemaic dynasty granted plots of land to all veteran Greek soldiers who would settle within the borders of the empire. These veterans and their decendants would then be called upon by the pharaoh to serve in the imperial army during times of war or rebellion. The pockets of Greek culture created by these settlements would influence and give shape to the traditional native Egyptian culture, creating a unique syncretism we today call the Hellenistic culture.
```
Comment (TheShadowOfHassen) [inline 0-ad-history-mod-package/simulation/templates/structures/ptol/military_colony.xml:12]:
```suggestion
        The Greco-Macedonian pharaohs of the Ptolemaic dynasty granted plots of land to all veteran Greek soldiers who would settle within the borders of the empire. These veterans and their descendants would then be called upon by the pharaoh to serve in the imperial army during times of war or rebellion. The pockets of Greek culture created by these settlements would influence and give shape to the traditional native Egyptian culture, creating a unique syncretism we today call the Hellenistic culture.
```

**Comment by indoptogopt** [comment]
Seems error-free.

**Pull request merged**